features/lorawan/lorastack/phy/LoRaPHYCN470.cpp

```diff
@@ -450,7 +450,7 @@ uint8_t LoRaPHYCN470::link_ADR_request(adr_req_params_t *params,
                                        uint8_t *nb_bytes_parsed)
 {
     uint8_t status = 0x07;
-    link_adr_params_t adr_settings;
+    link_adr_params_t adr_settings = {};
```

Comment:
That's not legal syntax C++03 syntax (I think), and shouldn't be required if the rest of the fix is good.